other-docs/guides/upgrading/v3.md

```diff
@@ -0,0 +1,143 @@
+
+Next run `composer update` to complete the upgrade. You should commit the updated `composer.json` and `composer.lock` files.
+
+
+## Breaking Changes
+
+### Media
+
+Intermediate image size files are no longer created but handled entirely by Tachyon if `modules.media.smart-media` is enabled in your config, this is on by default. This is only a breaking change if you have code that relies on fetching those files (not the original or full size image) directly by their path.
+
+### Block Editor
+
+The updated Block Editor in WordPress 5.3 has many updates to the underlying javascript packages such as React itself and features of the `@wordpress/data` package. If you make use of helpers like `wp.data.withDispatch` you should test this update thoroughly.
+
+
+## Headline Features
+
+### WordPress 5.3 "Kirk"
+
+The latest CMS Module brings with it WordPress version 5.3. Some of the highlights are:
+
+- Large image files are stored separately as the original and a web optimised full size version is generated for use on the website to improve performance
+- The Block Editor has many accessibility improvements and enhanced layout features
+- Automatic image rotation
+- Administrator email verification
+- PHP 7.4 Support
+
+You can [find the full WordPress 5.3 release notes here](https://wordpress.org/news/2019/11/kirk/).
+
+
+### Zero Configuration PHPUnit Testing
+
+Writing and runnning PHPUnit tests for your application is now a simple process. Add your test classes to a directory called `tests` in your project root directory and run `composer dev-tools phpunit`.
+
+Tests are run on the local environment whether it is [Local Server](docs://local-server) or [Local Chassis](docs://local-chassis) and as such provide full integration testing capabilities to match production with S3, ElasticSearch, caching and Tachyon.
+
+This feature provides flexibility for more complex use cases, check out the [unit testing documentation and examples here](docs://dev-tools/testing-with-phpunit.md).
+
+
+### Browser Security
+
+Good security is a growing concern online and so the [browser security](docs://security/browser.md) feature provides a configurable solution for securing static assets and cross origin behaviour. Supported features include:
+
+- Automatic Sub Resource Integrity hashes for 1st party scripts with 3rd party support via filters
+- Configurable `XSS-Protection` and `X-Frame-Options` headers
+- Content Security Policy support, with smart defaults and highly configurable
+
+
+## Other Developer Features and APIs
+
+### Core Module Loading (Core Module)
+
+Modules were previously loaded via composer's autoload functionality however this approach was less than ideal as the entrypoint files contained side-effects and required a `function_exists()` check at the start of the file to prevent certain composer commands breaking.
```

Comment:
```suggestion
Modules were previously loaded via Composer's autoload functionality, however this approach was limiting, as the entrypoint files contained side-effects and required a `function_exists()` check at the start of the file to prevent non-Altis CLI commands from breaking.
```